testcases/common/common.c

```diff
@@ -1237,21 +1240,21 @@ CK_BBOOL do_GetFunctionList(void)
     if (pkcs11lib == NULL)
         goto ret;
 
-    *(void **)(&pfoo) = dlsym(pkcs11lib, "C_GetFunctionList");
-    if (pfoo == NULL)
+    *(void **)(&getfunclist) = dlsym(pkcs11lib, "C_GetFunctionList");
```

Comment:
Maybe a dump question: why are you casting the variable here and not the return value of the dlsym() call? I think it would not make any difference, but for me it looks a bit uncommon.

Same is true for all other dlsym() calls.

Reply:
ok, make sense. Thanks a lot! So we should leave it as is.